### PR TITLE
Update wallet invocation section to mention Digital Credentials API

### LINKED ISSUE #87: wallet invocation ''no specific authorization_endpoint" option is under defined
In the wallet invocation section ( https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#section-7 ) this is one of the options:

> no specific authorization_endpoint, user scanning a QR code with Authorization Request using a manually opened Wallet, instead of an arbitrary camera application on a user-device (neither custom URL scheme nor Universal/App link is used)

This isn't very clear. I think it means you treat the Authorization Endpoint as being an empty string and hence the QR code contains  something like:

```
?response_type=vp_token
    &client_id=https%3A%2F%2Fclient.example.org%2Fcb
    &redirect_uri=https%3A%2F%2Fclient.example.org%2Fcb
    &presentation_definition=...
    &nonce=n-0S6_WzA2Mj
```

However if we want this option to be interoperable then we probably need to be more specific and explicitly say we mean that the Authorization Endpoint url is treated as an empty string in this case.

### LINKED ISSUE #262: Should Appendix A be referenced in section 7 as an option for Wallet Invocation?
[Section 7](https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#name-wallet-invocation) goes over options that Verifiers have to invoke wallets. Since [Appendix A](https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#name-openid4vp-profile-for-the-w) is one of those mechanisms, is there a reason it should not be noted there?

### PR DESCRIPTION
**Pull request by jogu**
Also remove the 'no specific authorization endpoint' method that was never clearly specified and no one seems to be using.

closes #87
closes #262

### CODE REVIEW

File: openid-4-verifiable-presentations-1_0.md [hunk 1]
```diff
@@ -1242,8 +1242,11 @@ Note: Some of the processing rules of the Presentation Definition and the Presen
 The Verifier can use one of the following mechanisms to invoke a Wallet:
 
 - Custom URL scheme as an `authorization_endpoint` (for example, `openid4vp://` as defined in (#openid4vp-scheme))
-- Domain-bound Universal Links/App link as an `authorization_endpoint`
-- no specific `authorization_endpoint`, End-User scanning a QR code with Authorization Request using a manually opened Wallet, instead of an arbitrary camera application on a user-device (neither custom URL scheme nor Universal/App link is used)
+- URL (including Domain-bound Universal Links/App link) as an `authorization_endpoint`
+
+For a cross device flow, either of the URL options MAY be presented as a QR code for the End-User to scan using a wallet or an arbitrary camera application on a user-device.
+
+The Wallet can also be invoked from the web or a native app using the Digital Credentials API as described in (#dc_api).
```
Comment (leecam) [inline openid-4-verifiable-presentations-1_0.md:1249]: I'd put this one first and if possible word it so it sounds like the preferred option :)

File: openid-4-verifiable-presentations-1_0.md [hunk 2]
```diff
@@ -1553,12 +1556,13 @@ However, the Response Mode `direct_post` is susceptible to such an attack as the
...
 
 Note that this protection technique is not applicable to cross-device scenarios because the browser used by the wallet will not have the original session.
-It is also not applicable in same-device scenarios if the wallet uses a browser different from the one used on the presentation request (e.g. device with multiple installed browsers), because the original session will also not be available there.
+It is also not applicable in same-device scenarios if the Wallet uses a browser different from the one used on the presentation request (e.g. device with multiple installed browsers), because the original session will also not be available there. (#dc_api) provides an alternative Wallet invocation method using web/appplatform APIs that avoids many of these issues.
```
[the remark this answers is not in the document]
Comment (jogu) [reply openid-4-verifiable-presentations-1_0.md:1559]: Thanks Oliver... no idea how I managed that!

File: openid-4-verifiable-presentations-1_0.md (hunk 1 above)
Comment (jogu) [reply openid-4-verifiable-presentations-1_0.md]: I think we might get some pushback against essentially recommending it so I'd rather consider that separately. Opened https://github.com/openid/OpenID4VP/issues/361 for that.

I think we can be clearer about the advantages though - @leecam @timcappalli could you check the new text please? Suggestions to improve it are welcome!

File: openid-4-verifiable-presentations-1_0.md [hunk 3]
```diff
@@ -1242,8 +1242,11 @@ Note: Some of the processing rules of the Presentation Definition and the Presen
 The Verifier can use one of the following mechanisms to invoke a Wallet:
 
 - Custom URL scheme as an `authorization_endpoint` (for example, `openid4vp://` as defined in (#openid4vp-scheme))
-- Domain-bound Universal Links/App link as an `authorization_endpoint`
-- no specific `authorization_endpoint`, End-User scanning a QR code with Authorization Request using a manually opened Wallet, instead of an arbitrary camera application on a user-device (neither custom URL scheme nor Universal/App link is used)
+- URL (including Domain-bound Universal Links/App link) as an `authorization_endpoint`
+
+For a cross device flow, either of the URL options MAY be presented as a QR code for the End-User to scan using a wallet or an arbitrary camera application on a user-device.
```
Comment (Sakurann) [inline openid-4-verifiable-presentations-1_0.md:1247]:
```suggestion
For a cross device flow, either of the above options MAY be presented as a QR code for the End-User to scan using a wallet or an arbitrary camera application on a user-device.
```
nit, but took me a while to process `URL options`

File: openid-4-verifiable-presentations-1_0.md [hunk 4]
```diff
@@ -1242,8 +1242,11 @@ Note: Some of the processing rules of the Presentation Definition and the Presen
...
+
+The Wallet can also be invoked from the web or a native app using the Digital Credentials API as described in (#dc_api), which provides security benefits over custom schemes (see (#session_fixation)) whilst also having privacy and user experience benefits (particularly in the cases where a user has multiple wallets).
```
Comment (Sakurann) [inline openid-4-verifiable-presentations-1_0.md:1249]: we describe the benefits of the DC API in detail in the DC API section... 
```suggestion
The Wallet can also be invoked from the web or a native app using the Digital Credentials API as described in (#dc_api). As described in detail in (#dc_api), DC API provides privacy, security (see (#session_fixation)), and user experience benefits (particularly in the cases where a user has multiple wallets) over custom schemes.
```

Comment (jogu) [reply openid-4-verifiable-presentations-1_0.md:1249]: Thanks! I don't think we should mention custom schemes though, a https deep link also has security disadvantages particularly when done cross-device.

```suggestion
The Wallet can also be invoked from the web or a native app using the Digital Credentials API as described in (#dc_api). As described in detail in (#dc_api), DC API provides privacy, security (see (#session_fixation)), and user experience benefits (particularly in the cases where a user has multiple wallets).
```

File: openid-4-verifiable-presentations-1_0.md (hunk 1 above)
Comment (Sakurann) [reply openid-4-verifiable-presentations-1_0.md]: we have a whole paragraph on the benefits that Tim helped re-write here: https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#appendix-A-4

**Review by Sakurann — APPROVED** [review]

File: openid-4-verifiable-presentations-1_0.md (hunk 1 above)
Comment (jogu) [reply openid-4-verifiable-presentations-1_0.md]: I've committed Kristina's suggestion to just refer to the existing text now.

**Review by bc-pi — APPROVED** [review]

**Review by c2bo — APPROVED** [review]

File: openid-4-verifiable-presentations-1_0.md [hunk 5]
```diff
@@ -1242,8 +1242,11 @@ Note: Some of the processing rules of the Presentation Definition and the Presen
 The Verifier can use one of the following mechanisms to invoke a Wallet:
 
 - Custom URL scheme as an `authorization_endpoint` (for example, `openid4vp://` as defined in (#openid4vp-scheme))
-- Domain-bound Universal Links/App link as an `authorization_endpoint`
-- no specific `authorization_endpoint`, End-User scanning a QR code with Authorization Request using a manually opened Wallet, instead of an arbitrary camera application on a user-device (neither custom URL scheme nor Universal/App link is used)
+- URL (including Domain-bound Universal Links/App link) as an `authorization_endpoint`
+
+For a cross device flow, either of the above options MAY be presented as a QR code for the End-User to scan using a wallet or an arbitrary camera application on a user-device.
+
+The Wallet can also be invoked from the web or a native app using the Digital Credentials API as described in (#dc_api). As described in detail in (#dc_api), DC API provides privacy, security (see (#session_fixation)), and user experience benefits (particularly in the cases where a user has multiple wallets).
```
[the remark this answers is not in the document]
Comment (jogu) [reply openid-4-verifiable-presentations-1_0.md:1249]: That's where I was trying to go with the session_fixation reference and adding the mention of the Browser API solution to the session fixation section. Do you think that hasn't achieved it?

File: openid-4-verifiable-presentations-1_0.md [hunk 6]
```diff
@@ -1242,8 +1242,11 @@ Note: Some of the processing rules of the Presentation Definition and the Presen
 The Verifier can use one of the following mechanisms to invoke a Wallet:
 
 - Custom URL scheme as an `authorization_endpoint` (for example, `openid4vp://` as defined in (#openid4vp-scheme))
-- Domain-bound Universal Links/App link as an `authorization_endpoint`
-- no specific `authorization_endpoint`, End-User scanning a QR code with Authorization Request using a manually opened Wallet, instead of an arbitrary camera application on a user-device (neither custom URL scheme nor Universal/App link is used)
+- URL (including Domain-bound Universal Links/App link) as an `authorization_endpoint`
+
+For a cross device flow, either of the above options MAY be presented as a QR code for the End-User to scan using a wallet or an arbitrary camera application on a user-device.
+
+The Wallet can also be invoked from the web or a native app using the Digital Credentials API as described in (#dc_api). As described in detail in (#dc_api), DC API provides privacy, security (see (#session_fixation)), and user experience benefits (particularly in the cases where a user has multiple Wallets).
```
Comment (paulbastian) [inline openid-4-verifiable-presentations-1_0.md:1249]: I'd prefer putting DC API into the enumeration but that's just nitpicking

**Review by paulbastian — APPROVED** [review]

File: openid-4-verifiable-presentations-1_0.md (hunk 6 above)
Comment (Sakurann) [reply openid-4-verifiable-presentations-1_0.md:1249]: let's do that when DC API is ready for production and we are changing this text to recommend it.

**Pull request merged**